sim/src/main/makefrag/firesim/Makefrag

```diff
@@ -101,12 +107,12 @@ run-verilator-debug: $(verilator_debug)
 
 run-vcs: $(vcs)
 	cd $(dir $<) && \
-	$(vcs) +permissive $(COMMON_SIM_ARGS) $(MIDAS_LEVEL_SIM_ARGS) $(EXTRA_SIM_ARGS) +permissive-off $(abspath $(SIM_BINARY)) \
+	$(vcs) +permissive $(vcs_args) $(COMMON_SIM_ARGS) $(MIDAS_LEVEL_SIM_ARGS) $(EXTRA_SIM_ARGS) +permissive-off $(abspath $(SIM_BINARY)) \
```

Comment:
Can you add these args to other MIDAS-level simulation invocations in the other Makefrags?